pkg/webhook/policy.go

```diff
@@ -115,15 +117,25 @@ func (h *validationHandler) Handle(ctx context.Context, req atypes.Request) atyp
 	resp, err := h.opa.Review(ctx, req.AdmissionRequest)
 	if err != nil {
 		log.Error(err, "error executing query")
-		return admission.ValidationResponse(false, err.Error())
+		vResp := admission.ValidationResponse(false, err.Error())
+		if vResp.Response.Result == nil {
+			vResp.Response.Result = &metav1.Status{}
+		}
+		vResp.Response.Result.Code = http.StatusInternalServerError
+		return vResp
 	}
 	res := resp.Results()
 	if len(res) != 0 {
 		var msgs []string
 		for _, r := range res {
 			msgs = append(msgs, r.Msg)
 		}
-		return admission.ValidationResponse(false, strings.Join(msgs, "\n"))
+		vResp := admission.ValidationResponse(false, strings.Join(msgs, "\n"))
+		if vResp.Response.Result == nil {
+			vResp.Response.Result = &metav1.Status{}
+		}
+		vResp.Response.Result.Code = http.StatusForbidden
```

Comment:
@maxsmythe 

Don't you want to send back 422 (Unprocessable entity) instead of 403 (Forbidden)

403 is what is sent back in an RBAC/AUTH failure and this is going to confuse the living daylights out of users. 

We use 422 in a number of other places to indicate validation failures:

e.g. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto#L728